pkg/provision/storage/shared.go

```diff
@@ -61,7 +61,6 @@ func WorkspaceNeedsStorage(workspace *dw.DevWorkspaceTemplateSpec) bool {
 }
 
 func getPVCSpec(name, namespace string, storageClass *string, size resource.Quantity) (*corev1.PersistentVolumeClaim, error) {
-
```

Comment:
@mkuznyetsov sorry for nit-pick, but could you please also restore this newline so that the diff is a bit cleaner?

Reply:
If possible, please re-add this newline.